devel/depot-tools-devel/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                depot-tools-devel
+
+version             20200530
+categories          devel
+platforms           darwin
+license             Permissive
+
+description         Tools for working with Chromium development.
+long_description    {*}${description}
+
+homepage            https://chromium.googlesource.com/chromium/tools/depot_tools
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+fetch.type          git
+git.url             ${homepage}.git
+git.branch          99044852de36333f88644adfd074189c05eaec12
```

Comment:
can this not use the `github` PG by setting `github.homepage`?

Reply:
Just tried that; seems like overriding `github.homepage` is not enough:

```
--->  Fetching distfiles for depot-tools-devel
DEBUG: Executing org.macports.fetch (depot-tools-devel)
DEBUG: Executing: /usr/bin/git clone --progress https://github.com/chromium/depot-tools.git /opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_devel_depot-tools-devel/depot-tools-devel/work/depot-tools-20200530 2>&1
DEBUG: system: /usr/bin/git clone --progress https://github.com/chromium/depot-tools.git /opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_devel_depot-tools-devel/depot-tools-devel/work/depot-tools-20200530 2>&1
Cloning into '/opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_devel_depot-tools-devel/depot-tools-devel/work/depot-tools-20200530'...
Username for 'https://github.com':
```

I feel that it's a lot cleaner and more straightforward to simply have the Portfile succinct as it is, as compared to having to fight with and override a lot of variables and behavior in the `github` portgroup.

Reply:
I haven't looked in detail but would suspect it should be possible to use the PortGroup. The thing I personally don't really like about how it is currently is that there are no checksums and that the guide appears to discourage the use of `fetch.type git`. But also, I don't have a immediate solutions, perhaps someone else will provide some insight.

Reply:
The reason there are no sums is because this is a git repository checkout.

The destination, googlesource.com does not provide a static binary whose checksums can be depended on not to change. As a result we use purely the git checkouts. I am aware that checksums would be the best policy; we could provide them even without the `github` portgroup. Unfortunately we cannot use them reliably in this context.